File: component_gank/src/main/runalone/java/com/baiiu/gank/runalone/GankTestActivity.java
package com.baiiu.gank.runalone;

import android.os.Bundle;
import com.baiiu.common.base.BaseActivity;
import com.baiiu.gank.R;

/**
 * author: baiiu
 * date: on 17/10/31 13:25
 * description:
 */
public class GankTestActivity extends BaseActivity {
    @Override public int provideLayoutId() {
        return R.layout.activity_test;
    }

    @Override protected void initOnCreate(Bundle savedInstanceState) {
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.container, GankListFragment.instance(), GankListFragment.class.getName())
                .commitAllowingStateLoss();
    }
}

File: runalone/gank/src/main/java/com/baiiu/gank/app/GankTestActivity.kt
package com.baiiu.gank.app

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.baiiu.gank.GankListFragment

class GankTestActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        supportFragmentManager.beginTransaction()
                .replace(R.id.container, GankListFragment(), GankListFragment::class.java.getName())
                .commitAllowingStateLoss()

    }
}
